packages/expo-sensors/src/__tests__/mocks/MockNativeSensorModule.ts

```diff
@@ -0,0 +1,7 @@
+export default class MockNativeSensorModule {
```

Comment:
I could be mistaken but I believe the mocks might differ between iOS and Android for event emitting modules. I'd double check on a real device to see whether we need separate mocks for each platform.

Reply:
The point of this PR is strictly to convert the tests to the new system.